consensus/application.go

```diff
@@ -957,7 +942,7 @@ func (au *ApplyUpdate) UnmarshalJSON(b []byte) error {
 	au.fces = js.FileContractElementDiffs
 	au.v2fces = js.V2FileContractElementDiffs
 	au.aes = js.AttestationElements
-	au.cie = js.ChainIndexElement
+	au.cie = js.ChainIndexElement.Move()
```

Comment:
Don't we have to call `Move` in a for loop on anything with a state element here?